script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt

```diff
@@ -717,27 +766,32 @@ Senpai leans forward.
 [sha:16dc933471bdb43c13005809a0bf1822f13a4411]{
 -- Page 292, Offset 34041.
 -- また走りこんで来るのは明白だが、俺には先輩の姿を視認する事さえ難しい。
-She's clearly going to rush over again, but I can't track her movements with my eyes properly.
+She's clearly going to rush me again,
+ but I don't stand a chance at tracking her movements.
 }
 [sha:93e75415aa53664985cc545c3778b9475286b04b]{
 -- Page 293, Offset 34042.
 -- ……逃げるしか手段がない。
-...I have no choice but to run.
+I have no choice but to run...
 }
 [sha:f3d28190c7afff1999227d02c1d45369be0e17ea]{
 -- Page 293, Offset 34043.
 -- 幸い、距離はさっきより離れている。このまま全力で走れば瓦礫の谷間に逃げこめる。
-Luckily, we're further apart now than we were a while ago. If I run at full speed, I should be able to retreat into the valley of rubble.
+Luckily, we're farther apart now than we were before.
+ If I run as fast as I can, I should be able to retreat into the valley of rubble.
```

Comment:
Both of them kinda have the problem of making it sound like there's a big ravine/gully/what have you, which there really isn't. It seems more like it's just the jumble of debris that's still in large pieces / affords some hidey holes. 
Maybe just drop the 谷間?
> retreat amongst the rubble.

Or invert it, since I feel like that just makes more sense at face value
> retreat into the mountains/mounds/piles of rubble.

Reply:
It's the remnants of the underground structures there, so its probably more extensive than you'd think. That said, any suggestion is fine by me, including my first one. Valley just sounds a bit too "place name" for me.